dash-renderer/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
+    </head>
+    <body>
+        <div id="react-entry-point"></div>
+    </body>
+
+    <script type="text/javascript">
+
+    </script>
+
+    <script type="text/javascript" src="build/bundle.js"></script>
+</html>
```

Comment:
what is this file for?

Reply:
moved together from dash-renderer, untouched chris index.html since 2017 :)